cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/deployment/AbstractUndeployCommand.java

```diff
@@ -225,4 +227,8 @@ private static ModelNode buildRequest(CommandContext ctx, String name,
         }
         return composite;
     }
+
+    private boolean isDisableAll() {
```

Comment:
by checking the name argument == "*" you have the information that the command is disable "all".